crates/goose/src/agents/extension_manager.rs

```diff
@@ -111,6 +111,8 @@ async fn child_process_client(
 ) -> ExtensionResult<McpClient> {
     #[cfg(unix)]
     command.process_group(0);
+    #[cfg(windows)]
+    command.creation_flags(0x08000000); // CREATE_NO_WINDOW flag: prevents console window from opening
```

Comment:
Nice. rather than the comment (he said yelling at the clouds), can we just introduce a named constant that says this?